packages/calling-component-bindings/src/participantListSelector.ts

```diff
@@ -30,14 +30,11 @@ const convertRemoteParticipantsToParticipantListParticipants = (
           const isScreenSharing = Object.values(participant.videoStreams).some(
             (videoStream) => videoStream.mediaStreamType === 'ScreenSharing' && videoStream.isAvailable
           );
-          return memoizeFn(
-            toFlatCommunicationIdentifier(participant.identifier),
-            participant.displayName,
-            participant.state,
-            participant.isMuted,
+          return memoizeFn({
```

Comment:
careful changing this from a set of parameters to an object. It's passed into a memoize function, but it is creating a new object everytime the memoizeFn is called - will this still be memoized?

Reply:
OK I reverted back to a set of parameters. I have to create a separate beta version of the function since conditional compilation does not work if the new function parameter is a beta type.